src/Transform.jl

```diff
@@ -1,5 +1,12 @@
 module Transform
 
-# Write your package code here.
+using Tables
+
+export Transformation, Power
```

Comment:
I wouldn't mind seeing a functional form of all of our transforms for eager evaluation on the repl. For example, you can do `impute(data, LOCF(...))` or `Impute.locf(data; kwargs...)` which is kinda nice. I could see a `@transform` macro that also generates this functional form along with a kwarg constructor.

Reply:
Yeah I agree, Distances does this as well I believe, e.g. `euclidean`, which would be quite convenient.